company.el

```diff
@@ -2521,11 +2540,18 @@ inserted."
       (when company-candidates
         (setq this-command 'company-complete-common)))))
 
-(defun company-complete-number (n)
+(define-obsolete-function-alias
+ 'company-complete-number
+ 'company-complete-quick-access
+ "0.10.0")
+
+(defun company-complete-quick-access (n)
   "Insert the Nth candidate visible in the tooltip.
-To show the number next to the candidates in some backends, enable
-`company-show-numbers'.  When called interactively, uses the last typed
-character, stripping the modifiers.  That character must be a digit."
+
+When called interactively, uses the last typed
+character, stripping the modifiers.  That character must be a digit.
+To enable a quick-access column for some backends, use
```

Comment:
Why "for some backends"?

Reply:
https://github.com/company-mode/company-mode/blob/d488122d18645a20a596490e3551194aecdea08a/company.el#L2527

"in some backends" is in the master (and been there for many years), so I hesitated about it and decided not to remove. Can it be safely removed?

Reply:
Guess I missed that during the review when this was added. :disappointed: 

Yes, remove it please.